bench-tps/src/cli.rs

```diff
@@ -557,6 +577,20 @@ pub fn parse_args(matches: &ArgMatches) -> Result<Config, &'static str> {
         args.client_node_id = Some(client_node_id);
     }
 
+    if let Some(values) = matches.values_of("load_accounts_from_address_lookup_table") {
```

Comment:
I think my above suggestion on splitting this into 2 args also makes this simpler wrt parsing.